app/src/main/java/io/webthings/app/templates/SourceTemplate.kt

```diff
@@ -0,0 +1,48 @@
+package io.webthings.app.templates
+
+import android.content.Context
+import android.widget.Toast
+
+abstract class SourceTemplate(val context: Context) {
+    /*
+     * Abstract class that will define a source of things
+     * all source module will need to inherit from this class
+     */
```

Comment:
```suggestion
    /**
     * Abstract class that will define a source of things
     * all source module will need to inherit from this class
     */
abstract class SourceTemplate(val context: Context) {
```

Comments should be above the line you are commenting.
Also it looks like this should a java doc comment (`**` instead of `*`)